### PR TITLE
Added some something basic to expose the standalone cli.

### PR DESCRIPTION
**Pull request by markhibberd**

### CODE REVIEW

**Comment by markhibberd** [comment]
@charleso @nhibberd

File: src/Ambiata/Cli/Standalone.hs [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  runAWST privileged UploadAwsError . bimapEitherT UploadS3Error id $
+    S3.uploadWithMode S3.Overwrite f target
+
+
+-- |
+-- Execute a program and upload its standard out as the specified filename.
+--
+-- Note: This is not a great idea, it will be _slow_ and potentially
+-- far less reliable than uploading a file directly, but may be
+-- convenient for small data.
```
Comment (markhibberd) [inline src/Ambiata/Cli/Standalone.hs]: Note well.

**Comment by charleso** [comment]
👍

File: src/Ambiata/Cli/Standalone.hs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+renderDownloadError err =
+  case err of
+    DownloadApiError e ->
+      mconcat ["There was an error contacting the Ambiata API - ", renderApiError e]
+    DownloadAwsError e ->
+      mconcat ["There was an error contacting the Amazon API - ", renderError e]
+    DownloadAddressNotAvailable a ->
+      mconcat ["The specified download is no longer available for download: ", S3.addressToText a]
+    DownloadS3DownloadError e ->
+      mconcat ["The requested download could not be complete: ", S3.renderDownloadError e]
```
Comment (olorin) [inline src/Ambiata/Cli/Standalone.hs]: s/e:/ed:/

**Pull request merged**